src/state.h

```diff
@@ -30,6 +30,9 @@
 // pixels.
 #define BISECT_MAX_HISTORY 16
 
+// Split history of up to a resolution of 65536x65536
+#define SPLIT_MAX_HISTORY 16
```

Comment:
That's too small.

The `split` mode divides the areas twice as slowly as the `bisect` mode because we're only bisecting one axis at a time instead of both. Assuming the same number of splits each way, we can only handle a 256&times;256px maximum resolution here.

For example, if I have a resolution of 1920x1080, I can reach a width of 1px with 11 splits which only leaves 5 splits for the height which is not enough to reach a pixel height area.

```suggestion
#define SPLIT_MAX_HISTORY 32
```

Reply:
You're right. The current value is not enough. For a 4k resolution, the value should be  log2(3840) + log2(2160) ~=  23.

32 should accommodate a resolution of 65536x65536.